README.md

```diff
@@ -44,6 +44,14 @@ Spyder may also be used as a PyQt4/PySide extension library (module
 `spyderlib`). For example, the Python interactive shell widget used in
 Spyder may be embedded in your own PyQt4/PySide application.
 
+## Running from source
+
+It is possible to run Spyder from the source code checkout, given
+that you have PyQt or PySide installed. Just run:
```

Comment:
Why the reference to PyQt and PySide here ? Spyder also needs numpy, shinx, and others, right ?